index.html

```diff
@@ -0,0 +1,214 @@
+      </nav>
+    </header>
+
+    <section class="main-mobile-section">
+      <div class="ellipse1"></div>
+      <div class="ellipse2"></div>
+
+      <h1>Gideon Dadi</h1>
+      <h3>Software Developer</h3>
+      <div class="advert-text">
+        <p>
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+
+      <hr class="vector4" />
+
+      <div class="tech-experience">
+        <ul class="language">
+          <li><h2>Language</h2></li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>CSS</li>
+          <li>Html</li>
+        </ul>
+
+        <ul class="frameworks">
+          <li><h2>Frameworks</h2></li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+
+        <ul class="skills">
+          <li><h2>Skills</h2></li>
+          <li>CodeKit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+
+        <hr class="vector5" />
+      </div>
+
+      <ul class="social-media-icons">
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector4.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector3.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector2.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector5.png" alt="" />
+          </a>
+        </li>
+      </ul>
+
+      <div class="work-arrow">
+        <a href="#"> <img src="icons_images/Vector6.png" alt="" /> Work </a>
+      </div>
+
+      <h1 class="my-work-header">MY RECENT WORK</h1>
+      <div class="vector6"></div>
+
+      <div class="my-work">
+        <div class="my-work-1">
+          <div class="img-placeholder1"></div>
+          <h2 class="work-title1">Multi-Post Stories</h2>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+
+          <div class="tech-used-1">
+            css <img src="icons_images/line.png" alt="Line" /> html
+            <img src="icons_images/line.png" alt="Line" />bootstrap
+            <img src="icons_images/line.png" alt="Line" />Ruby
+          </div>
+
+          <button class="button" type="button">See Project</button>
+        </div>
```

Comment:
- [ ] great work up to here, your design looks awesome, but I think we can still make this better by adding some button action as per this image below, let's go and add that too 😄 , consider making the change on all similar buttons, please

<img width="417" alt="hover" src="https://user-images.githubusercontent.com/79058364/154757644-a5ce0deb-ebe8-431b-8333-c54482d66f73.png">